one-shot-device/src/configuration/configuration.ts

```diff
@@ -0,0 +1,16 @@
+import { ApiVersion, ManagerConfig, ClientConfig } from 'iota-is-sdk';
+import * as dotenv from 'dotenv';
+
+dotenv.config();
+
+export const defaultConfig: ClientConfig = {
+  apiKey: process.env.API_KEY,
+  baseUrl: process.env.API_URL,
+  apiVersion: ApiVersion.v01,
+};
+
+export const defaultManagerConfig: ManagerConfig = {
```

Comment:
seems like this is not used. Please remove it.

Reply:
Fixed.